src/pages/dar_application/ResearcherInfo_new.js

```diff
@@ -0,0 +1,499 @@
+                }
+                return option;
+              }
+            },
+          }),
+        ]),
+
+        div({className: 'dar-application-row'}, [
+          h(FormField, {
+            id: 'itDirector',
+            description: 'I certify that the individual listed below is my IT Director',
+            placeholder: 'IT Director',
+            title: '1.7 Information Technology (IT) Director',
+            validators: [FormValidators.REQUIRED],
+            ariaLevel: ariaLevel + 1,
+            onChange: ({key: name, value,}) => formFieldChange({name, value}),
+            defaultValue: itDirector
+          })
+        ]),
+
+        div({className: 'dar-application-row'}, [
+          div([
+            h(FormField, {
+              id: 'anvilUse',
+              type: FormFieldTypes.RADIO,
+              title: '1.8 Cloud Use Statement',
+              description: [span([
+                'Will you perform all of your data storage and analysis for this project on the ',
+                a({
+                  rel: 'noopener noreferrer',
+                  href: 'https://anvil.terra.bio/',
+                  target: '_blank'
+                }, ['AnVIL']),
+                '?'
+              ])],
+              options: [
+                { name: 'yes', text: 'Yes' },
+                { name: 'no', text: 'No' }
+              ],
+              validators: [FormValidators.REQUIRED],
+              ariaLevel: ariaLevel + 1,
+              orientation: 'horizontal',
+              onChange: ({key: name, value}) => {
+                const normalizedValue = value && value.selected === 'yes';
+                formFieldChange({name, value: normalizedValue});
+              },
+              defaultValue: { selected: anvilUse ? 'yes' : 'no' }
+            })
+          ])
+        ]),
+
+        div({className: 'dar-application-row'}, [
+          div({className: 'row no-margin'}, [
+            div({
+              isRendered: anvilUse === false,
+              className: 'computing-use-container',
+              style: {
+                backgroundColor: showValidationMessages && isCloudUseInvalid ? 'rgba(243, 73, 73, 0.19)' : 'inherit'
+              }
+            }, [
+              div({className: 'row no-margin'}, [
+                div({className: 'col-lg-12 col-md-12 col-sm-12 col-xs-12 rp-group'}, [
+                  h(FormField, {
+                    id: 'cloudUse',
+                    disabled: !isNil(darCode),
+                    validators: [FormValidators.REQUIRED],
+                    type: FormFieldTypes.CHECKBOX,
+                    toggleText: 'I am requesting permission to use cloud computing to carry out the research described in my Research Use Statement',
+                    defaultValue: cloudUse,
+                    ariaLevel: ariaLevel + 2,
+                    onChange: ({ key: name, value }) => formFieldChange({name, value})
+                  })
+                ])
+              ]),
```

Comment:
should this be below the following checkbox? It's a little bit confusing that the conditional fields are separated from what causes them to render. I think it would make sense to move this one down or the conditional fields up.

Reply:
Ah, good point (and something that we should've caught/adjusted with the old form as well). For this ticket the easiest thing to do is to render the local checkbox first, then the cloud computing. This way the cloud fields are, relatively speaking, grouped together while keeping the checkbox options and input fields separate.

Reply:
I just moved the checkboxes into the same parent group - But I dont know, I feel like it makes more sense to keep the entire radio group together than to spread it out (with the conditional renders between).

Reply:
I should've been more clear. I was just asking to switch the order of the checkboxes so that local is first and cloud computing is second. This way, when you do click on cloud computing, the text inputs to populate cloud computing information will render below it.